src/CONST.js

```diff
@@ -420,12 +419,12 @@ const CONST = {
         LARGE: 'large',
         DEFAULT: 'default',
     },
-
+    PHONE_MAX_LENGTH: 15,
     REGEX: {
         US_PHONE: /^\+1\d{10}$/,
         DIGITS_AND_PLUS: /^\+?[0-9]*$/,
         PHONE_E164_PLUS: /^\+?[1-9]\d{1,14}$/,
-        PHONE_WITH_SPECIAL_CHARS: /^[+]*[(]{0,1}[0-9]{1,3}[)]{0,1}[-\s\\./0-9]*$/,
+        PHONE_WITH_SPECIAL_CHARS: /^[+]*[(]{0,1}[0-9]{1,3}[)]{0,1}[-\s\\./0-9]{0,11}$/,
```

Comment:
I think this brings us to a max of 14 digits total, right? I think we want to bump this to `{0,12}`

Reply:
Sorry about this. Updated.